bin/xucli

```diff
@@ -27,6 +27,11 @@ require('yargs')
       describe: 'Path to the TLS certificate of xud',
       type: 'string',
     },
+    disabletls: {
```

Comment:
Why not set `tlscertpath` to `undefined` or just to an empty string to disable TLS instead of having an own field for it?

Reply:
I thought about this, my main reason to go with it's own option is for the easy `-d` arg. I don't know of a way to pass an empty string or `undefined` for a string argument, if you do then I'd be fine with that approach.